### ISSUE TITLE
Postproc multiyear optics

### ISSUE THREAD

**Issue by plazzari** [inogs/ogstm_postproc#1]
We opened PR https://github.com/inogs/ogstm_postproc/pull/2 to address this issue

**Output frequency for optics**

- Kd daily
- Rrs(0+) daily
- Light at the bottom multispectral daily
- Raw optical files (Ed,Es,Eu) 3D daily lossy compression when archived

**New features**

- Script for rrs calculation to be upgraded using lat as correction.
- Script to compute multispectral  light at the bottom for all components Ed, Es, Eu (at n+1 layer)
- Script to save Ed,Es,Eu with higher compression (5 significant digit)

**Comment by gbolzon** [comment]
With 5 significant digits compression for 3D files
```
3.2M  ave.19991101-12:00:00.Ed_0400.nc
 13M  ave.19991101-12:00:00.Es_0400.nc
 11M  ave.19991101-12:00:00.Eu_0400.nc
```

The resulting archive space is 1TB / year,  the total about 20TB.

With 4 significant digits we have 0.7TB/ year, the total about 14TB.
With 3 significant digits we have 0.5TB/ year, the total about 10TB.

(Original file 202 MB, with no-loss compression 30MB)


Here are these case of compression
```
/g100_scratch/userexternal/gbolzon0/NECCTON/testCA/run_link/wrkdir/POSTPROC/bin/archive/2digits/ave.19990501-12:00:00.Eu_0400.nc
/g100_scratch/userexternal/gbolzon0/NECCTON/testCA/run_link/wrkdir/POSTPROC/bin/archive/3digits/ave.19990501-12:00:00.Eu_0400.nc
/g100_scratch/userexternal/gbolzon0/NECCTON/testCA/run_link/wrkdir/POSTPROC/bin/archive/4digits/ave.19990501-12:00:00.Eu_0400.nc
/g100_scratch/userexternal/gbolzon0/NECCTON/testCA/run_link/wrkdir/POSTPROC/bin/archive/5digits/ave.19990501-12:00:00.Eu_0400.nc
```
Can be useful to compare them with the original one?
`/g100_scratch/userexternal/camadio0/Neccton_hindcast1999_2022/wrkdir/MODEL/AVE_FREQ_3/ave.19991101-12:00:00.Eu_0400.nc`

**Comment by CarolinaAmadio** [comment]
is it possible to save dcm , mld , nutricline for each of the 16 sub-basins? (metrics2d.py --> avescan )

**Comment by gbolzon** [comment]
> is it possible to save dcm , mld , nutricline for each of the 16 sub-basins? (metrics2d.py --> avescan )

Yes, it's possible @plazzari , @CarolinaAmadio , what frequency? Daily or weekly?

**Comment by plazzari** [comment]
I would process Ed,Es,Eu with daily frequency 2D surface values and bottom values, and 3D only around the 
BGC-Argo float wavelengths 380, 412, 490.

**Comment by gbolzon** [comment]
We opened PR #2 to address this issue

**Comment by gbolzon** [comment]
Merged in main branch
